app/src/main/res/values/strings.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
these strings should be merged with the ones in `common` we dont use strings in `main`